config/environments/development.rb

```diff
@@ -11,12 +11,14 @@
 
   config.action_cable.url = ENV.fetch('ACTION_CABLE_URL') { "wss://#{ENV['FQDN']}/cable" }
   config.action_cable.allowed_request_origins = [ /.+/ ]
+  config.assets.check_precompiled_asset = false
 
   # Do not eager load code on boot.
   config.eager_load = false
 
   # Show full error reports.
   config.consider_all_requests_local = true
+  config.assets.manifest = Rails.root.join('config/manifest.json')
```

Comment:
Is this still necessary?  Also, maybe not `config.assets.check_precompiled_asset = false` for development?

Reply:
No, and sure